extensions/Article/Entity/Article.xml

```diff
@@ -37,13 +37,6 @@
                description="COM_CONTENT_FIELD_ARTICLETEXT_DESC"
         />
 
-        <field name="author"
-               type="text"
-               label="COM_CONTENT_FIELD_CREATED_BY_ALIAS_LABEL"
-               description="COM_CONTENT_FIELD_CREATED_BY_ALIAS_DESC"
-               default=""
-        />
-
```

Comment:
In conjunction with a user component, the article's `author` field becomes the `author alias`, we know from current com_content, which *is* a text field.